cross/c_icap/PLIST

```diff
@@ -1,7 +1,7 @@
 bin:bin/c-icap
 bin:bin/c-icap-client
-bin:bin/c-icap-config
-bin:bin/c-icap-libicapapi-config
+rsc:bin/c-icap-config
```

Comment:
Best to keep with `bin:` prefix for files in `bin` dir. Even though these files can't be stripped right now, this may change with an update of c-icap.
Having to check PLISTS is tedious already, let's not make that worse by actually having to check if a file can be stripped or not on each update.